domains/client/domain-operator/src/tests.rs

```diff
@@ -1091,6 +1091,155 @@ async fn test_invalid_domain_extrinsics_root_proof_creation() {
     ferdie.produce_blocks(1).await.unwrap();
 }
 
+#[tokio::test(flavor = "multi_thread")]
+#[ignore]
+async fn test_valid_bundle_proof_generation_and_verification() {
```

Comment:
I would prefer this test is added in commit where the proof generation or verification is added instead of seperate commit

Reply:
I prefer to push unit test in the same commit that contains changes the unit test is testing, but push integration test in a separate commit because what it testing are changes introduced by multiple commits and this makes the integration test easier to operate with git cherry-pick/revert, etc.